ci/playbooks/edpm_build_images/edpm_build_images_content_provider_run.yaml

```diff
@@ -16,14 +16,15 @@
       ansible.builtin.command:
         chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
         cmd: >-
-          ansible-playbook ci/playbooks/edpm_build_images/edpm_build_images_content_provider.yaml
+          ansible-playbook -i localhost, -c local
+          ci/playbooks/edpm_build_images/edpm_build_images_content_provider.yaml
+          -e "@{{ ansible_user_dir }}/ci-framework-data/artifacts/parameters/zuul-params.yml"
```

Comment:
Maybe I missed this discussion already, but zuul params should be in the end, so jobs can override scenario vars.

Reply:
Good point, I'll address it

Reply:
thanks @viroel - wasn't sure if it was really intended or not.